Cargo.toml

```diff
@@ -23,3 +23,6 @@ glam = {version = "0.8", features = ["scalar-math"] }
 image = { version = "0.22", default-features = false, features = ["png_codec", "tga"] }
 megaui = { version = "=0.2.11" }
 macroquad_macro = { version = "0.1", path = "macroquad_macro" }
+
+[patch.crates-io]
```

Comment:
This probably should not be a part of this PR?

Reply:
Removed